content/applications/inventory_and_mrp/inventory/shipping_receiving/picking_methods/batch.rst

```diff
@@ -82,7 +77,6 @@ Once the transfer records are selected, click :guilabel:`Confirm` to confirm the
    :guilabel:`Operation Type`. The :guilabel:`Scheduled Date` is set to `August 11`.
```

Comment:
In the example admonition, I would make it a full sentence by adding a couple words at the beginning, e.g. "Here is a new batch transfer..."